components/HomePage/InfoSection.tsx

```diff
@@ -1,21 +1,24 @@
 import styled from 'styled-components'
 
-//todo: add dynamic content based on database class descriptions
-export default function InfoSection({ id, imgStart }) {
+interface InfoSectionProps {
+  id?: string
+  imgStart: boolean
+}
+export default function InfoSection({ id, imgStart }: InfoSectionProps) {
   return (
     <>
       <InfoContainer id={id}>
         <InfoWrapper>
           <InfoRow imgStart={imgStart}>
-            <Column imgStart={imgStart}>
+            <Column>
               <TextWrapper>
                 <TopLine>topline</TopLine>
                 <Heading>heading</Heading>
                 <Subtitle>subtitle</Subtitle>
               </TextWrapper>
             </Column>
-            <Column imgStart={imgStart}>
-              <ImgWrap imgStart={imgStart}>
+            <Column>
+              <ImgWrap>
                 <Img src="/shoe.jpeg" />
```

Comment:
Use relative paths. Better yet; `import` the image as a default import and use that in the `src`

e.g.
```javascript
import MyShoeImag from '../../assets/shoe.jpg

<Img src={MyShoeImag} />

```

Reply:
Also, give all images an `alt` tag (and maybe even a `title` tag as well).
Just keep in mind how to use `alt` and `title`. They shouldn't be the same. If there is no other way, than remove the `title`

Reply:
This is how nextjs recommends to serve static files. I'll add the alt and title. https://nextjs.org/docs/basic-features/static-file-serving

Reply:
Agreed! (Love it that you counter my argument with the docs 😄  👏  Proud)